From: Michael Biebl <biebl@debian.org>
Date: Tue, 10 Mar 2015 22:55:22 +0100
Subject: Don't explicitly link tcpflood against -lgcrypt

While oder GnuTLS versions do use gcrypt, we rely on GnuTLS to pull that
library dependency in.
---
 tests/Makefile.am | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/tests/Makefile.am b/tests/Makefile.am
index f792b44ae..982b33872 100644
--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -1414,9 +1414,6 @@ uxsockrcvr_LDADD = $(SOL_LIBS)
 tcpflood_SOURCES = tcpflood.c
 tcpflood_CPPFLAGS = $(PTHREADS_CFLAGS) $(GNUTLS_CFLAGS) $(RELP_CFLAGS)
 tcpflood_LDADD = $(SOL_LIBS) $(PTHREADS_LIBS) $(GNUTLS_LIBS) $(RELP_LIBS)
-if ENABLE_GNUTLS
-tcpflood_LDADD += -lgcrypt
-endif
 
 minitcpsrv_SOURCES = minitcpsrvr.c
 minitcpsrv_LDADD = $(SOL_LIBS)
